oak-search-elastic/pom.xml

```diff
@@ -221,6 +221,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <!-- Metrics -->
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-core</artifactId>
+    </dependency>
+
```

Comment:
`ElasticMetricHandler` uses the classes defined in `org.apache.jackrabbit.oak.stats`. Can you do the same?

Reply:
done

Reply:
Remove the dep if no longer required.